semcore/d3-chart/src/RadialTree.tsx

```diff
@@ -0,0 +1,768 @@
+import React from 'react';
+import { Component, sstyled, CProps, ReturnEl } from '@semcore/core';
+import createElement from './createElement';
+import uniqueIDEnhancement from '@semcore/utils/lib/uniqueID';
+import { transition } from 'd3-transition';
+import style from './style/radial-tree.shadow.css';
+import { shade } from '@semcore/utils/lib/color';
+
+const memoize = <T extends (...args: (string | number)[]) => any>(func: T): T => {
```

Comment:
this function used in Bubble chart, maybe you can move this code in d3-chart/utils ?